config/_ci.json

```diff
@@ -0,0 +1,3 @@
+{
+  "publish": null
```

Comment:
Unfortunately, `"publish": null` is a critical piece of configuration, because without it automatic publishing rules take effect and our Travis/Appveyor builds will attempt to publish releases by default.  You can read more in electron-builder's [How To Publish](https://www.electron.build/configuration/publish) docs.